client/bridge_port.go

```diff
@@ -7,11 +7,12 @@ import (
 
 // BridgePort defines port-in-bridge association
 type BridgePort struct {
-	Id        string `mikrotik:".id"`
-	Bridge    string `mikrotik:"bridge"`
-	Interface string `mikrotik:"interface"`
-	PVId      int    `mikrotik:"pvid"`
-	Comment   string `mikrotik:"comment"`
+	Id         string `mikrotik:".id"`
+	Bridge     string `mikrotik:"bridge"`
+	Interface  string `mikrotik:"interface"`
+	PVId       int    `mikrotik:"pvid"`
+	Comment    string `mikrotik:"comment"`
+	FrameTypes string `mikrotik:"frame-types"`
```

Comment:
This field has value `admit-all` by default, but [expected value](https://github.com/ddelnano/terraform-provider-mikrotik/blob/af8c34b1640e9b32618c7f84c40f575f13e69d97/client/bridge_port_test.go#L43) does not populate it and `TestBridgePort_basic` fails.

Reply:
This should be updated :)

Reply:
GithubActions will indicate success/failure once all conflicts are resolved.